packages/ckeditor5-style/tests/ui/stylegridview.js

```diff
@@ -108,4 +116,140 @@ describe( 'StyleGridView', () => {
 			expect( grid.element.lastChild ).to.equal( grid.children.last.element );
 		} );
 	} );
+
+	describe( 'render()', () => {
+		beforeEach( () => {
```

Comment:
This is used by a single test only, the rest does their own prep. Might be worth it to move this prep to the `it()` that needs it.